<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результаты олимпиады: Фильтрация</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>trophy.png</normaloff>trophy.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="schoolInput">
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="classInput">
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Узнать результаты</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>